CONTRIBUTING.md

```diff
@@ -29,6 +29,11 @@ If you wish to contribute code (features or bug fixes) please keep in mind the f
 - bug fix pull requests should be opened against `main` as the base branch
 - feature pull requests should be opened with `next` as the base branch
 
+If your pull request is still in progress, but you want get some early reviews before
```

Comment:
```suggestion
If your pull request is still in progress, but you want to get an earl review before
```

Reply:
Nit: I believe this should be placed in its own section, not in `Finding work`. Something like `Working with Pull Requests` maybe.

Reply:
updated.

Reply:
ping: https://github.com/Kong/kubernetes-ingress-controller/pull/4677#discussion_r1329856858

Reply:
ADDED the section and a simple guide of submitting PRs.